src/main/java/Main.java

```diff
@@ -0,0 +1,63 @@
+public class Main {
+    public static void main(String[] args) {
+        Figure[] figures = new Figure[1 + (int) (Math.random() * 10)];
+        //массив, содержащий от 1 до 10 фигур
+        for (int i = 0;i < figures.length;i++) {
+            figures[i] = randomFigure();
+        }
+        for (Figure figure : figures) {
+            System.out.println(figure.toString());
+        }
+    }
+
+    public static int randomSize() {
+        return 1 + (int) (Math.random() * 10); // делаю рандомный размер от 1 до 10
+    }
+    //генерирую рандомную фигуру
```

Comment:
method name is enough in this case, add comments whenever you have complicated logic and additional explanation is required

Reply:
deleted